uiowa/uiowa
# Concept: Grid layout options per layout region

# How to test
* `blt frontend`
* `blt ds`
* `drush @default.local uli /`
* Click 'Layout' tab
* Configure any section to see new options.
* After saving the section configuration and the page, inspect the section HTML in the page view.
* You should see classes added to the each region based on the grid setting you selected.
